index.xml

```diff
@@ -365,128 +366,7 @@ implementations of the specification (in alphabetical order).
-        <c>blake2s-24</c><c>0xb243</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-32</c><c>0xb244</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-40</c><c>0xb245</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-48</c><c>0xb246</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-56</c><c>0xb247</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-64</c><c>0xb248</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-72</c><c>0xb249</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-80</c><c>0xb24a</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-88</c><c>0xb24b</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-96</c><c>0xb24c</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-104</c><c>0xb24d</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-112</c><c>0xb24e</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-120</c><c>0xb24f</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-128</c><c>0xb250</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-136</c><c>0xb251</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-144</c><c>0xb252</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-152</c><c>0xb253</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-160</c><c>0xb254</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-168</c><c>0xb255</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-176</c><c>0xb256</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-184</c><c>0xb257</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-192</c><c>0xb258</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-200</c><c>0xb259</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-208</c><c>0xb25a</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-216</c><c>0xb25b</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-224</c><c>0xb25c</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-232</c><c>0xb25d</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-240</c><c>0xb25e</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-248</c><c>0xb25f</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-        <c>blake2s-256</c><c>0xb260</c><c>active</c><c><xref target="RFC7693">RFC 7693</xref></c>
-      </texttable>
+      &registry;
```

Comment:
Hmm, do you know if this change will break the browser-based rendering of the specification? Note that you can view a live version of the latest editor's draft here: https://w3c-dvcg.github.io/multihash/index.xml ... is this PR going to break that browser-based preview?

Reply:
I was not aware of that but the preview is incomplete anyway because it does not include references. I could extend the `.xsl` file to fix this.

Reply:
Yes, please do, if possible. I'm not paying too much attention to the *how*... just that we want the browser-based rendering to continue to work.